cmd/ci-operator/main.go

```diff
@@ -786,6 +788,29 @@ func overrideMultiStageParams(o *options) error {
 	return nil
 }
 
+// applyEnvOverrides processes environment variables with override prefixes and applies them to the test configurations.
+// It checks for environment variables that start with "OVERRIDE_" and applies them to the environment settings of each test.
+func applyEnvOverrides(o *options) {
+	for _, envVar := range os.Environ() {
+		parts := strings.SplitN(envVar, "=", 2)
+		if len(parts) != 2 {
+			continue
+		}
+		key, value := parts[0], parts[1]
+		if !strings.HasPrefix(key, "MULTISTAGE_PARAM_OVERRIDE_") {
```

Comment:
Move `strings.HasPrefix` at the top of this loop, before `strings.SplitN`

Reply:
An error occurs when the value of an environment variable contains MULTISTAGE_PARAM_OVERRIDE_ but the key does not contain MULTISTAGE_PARAM_OVERRIDE_.